ui/app/components/app/wallet-overview/eth-overview.js

```diff
@@ -0,0 +1,110 @@
+import React, { useContext } from 'react'
+import { useDispatch, useSelector } from 'react-redux'
+import classnames from 'classnames'
+import { useHistory } from 'react-router-dom'
+
+import Button from '../../ui/button'
+import Identicon from '../../ui/identicon'
+import { I18nContext } from '../../../contexts/i18n'
+import WalletOverview from './wallet-overview'
+import { SEND_ROUTE } from '../../../helpers/constants/routes'
+import { useMetricEvent } from '../../../hooks/useMetricEvent'
+import Tooltip from '../../ui/tooltip-v2'
+import UserPreferencedCurrencyDisplay from '../user-preferenced-currency-display'
+import { PRIMARY, SECONDARY } from '../../../helpers/constants/common'
+import { showModal } from '../../../store/actions'
+import { isBalanceCached, getSelectedAccount, getShouldShowFiat } from '../../../selectors/selectors'
+
+const EthOverview = () => {
+  const dispatch = useDispatch()
+  const t = useContext(I18nContext)
+  const metricsEvent = useMetricEvent()
+  const history = useHistory()
+  const balanceIsCached = useSelector(isBalanceCached)
+  const showFiat = useSelector(getShouldShowFiat)
+  const selectedAccount = useSelector(getSelectedAccount)
+  const { balance } = selectedAccount
+
+  return (
+    <WalletOverview
+      balance={(
+        <Tooltip position="top" title={t('balanceOutdated')} disabled={!balanceIsCached}>
+          <div className="eth-overview__balance">
+            <div className="eth-overview__primary-container">
+              <UserPreferencedCurrencyDisplay
+                className={classnames('eth-overview__primary-balance', {
+                  'eth-overview__cached-balance': balanceIsCached,
+                })}
+                value={balance}
+                type={PRIMARY}
+                ethNumberOfDecimals={4}
+                hideTitle
+              />
+              {
+                balanceIsCached ? <span className="eth-overview__cached-star">*</span> : null
+              }
+            </div>
+            {
+              showFiat && (
+                <UserPreferencedCurrencyDisplay
+                  className={classnames({
+                    'eth-overview__cached-secondary-balance': balanceIsCached,
+                    'eth-overview__secondary-balance': !balanceIsCached,
+                  })}
+                  value={balance}
+                  type={SECONDARY}
+                  ethNumberOfDecimals={4}
+                  hideTitle
+                />
+              )
+            }
+          </div>
+        </Tooltip>
+      )}
+      buttons={(
+        <>
+          <Button
+            type="secondary"
+            className="eth-overview__button"
+            onClick={() => {
+              metricsEvent({
+                eventOpts: {
```

Comment:
The `useMetricsEvent` api is a little different than just exposing a context function. You'd want to use that hook multiple times for each payload you want to use, it returns a memoized callback function that you can just pass to your click handler. 

eg:
https://github.com/MetaMask/metamask-extension/blob/8a0230f7437b8704e4289d534d1f5080e26c81c0/ui/app/hooks/useRetryTransaction.js#L32-L38

Reply:
Whoops! I'll fix this.

Reply:
All three spots have been updated.

Reply:
I used this incorrectly in the AssetList component as well apparently. Fixed in #8639